src/backend/src/Tippecanoe.jl

```diff
@@ -0,0 +1,14 @@
+module Tippecanoe
+export tippecanoe
+
+# NB: not a real package yet
+using tippecanoe_jll: tippecanoe
+
+function tippecanoe(source::String, outdir::String)
```

Comment:
I think this will cause an error? Or tippecanoe will only have the one method. If you want to add a method to the existing binary dep then that should be `import`.

I don't reallly like these little wrappers around the binary deps. I don't think they're clearer than just calling `run` directly.

In this case, I feel like this is a bad name as well. If we have it, it should be called `output_to_directory` or `make_tiles` or something.

Reply:
My preference is that we just use Tippecanoe_jll directly in whatever data-prep code we write.

Reply:
> I don't reallly like these little wrappers around the binary deps. I don't think they're clearer than just calling run directly.

Yeah, I think I agree for `tippecanoe`. I just did the same as I did for `ogr2ogr` without thinking.

I think the `ogr2ogr` interface makes a bit more sense as the CLI is ... weird (hello `/vsistdout/`). It could one day become a PR to ArchGDAL or perhaps GeoJSON (via Requires.jl?) to allow the direct import of shapefiles.